src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Document/Document.ts

```diff
@@ -262,7 +265,8 @@ export var postEdit = (
     var paragraphVersion : RIParagraphVersion;
 
     _.forEach(scope.data.paragraphs, (paragraph : IParagraph, index : number) => {
-        if (index >= oldParagraphVersions.length) {
+        // currently, if a paragraph has been deleted, it doesn't get posted at all.
+        if (index >= oldParagraphVersions.length && !paragraph.deleted) {
```

Comment:
I think it would be easier to read if you wrap an `if (!paragraph.deleted)` around the whole block.